src/bp/core/services/dialog/instruction/strategy.ts

```diff
@@ -110,7 +110,14 @@ export class ActionStrategy implements InstructionStrategy {
       throw new Error(`Action "${actionName}" has invalid arguments (not a valid JSON string): ${argsStr}`)
     }
 
-    args = _.mapValues(args, value => renderRecursive(value, { event }))
+    const actionArgs = {
+      event,
+      user: _.get(event, 'state.user') || {},
```

Comment:
`_.get(event, 'state.user') || {} -> _.get(event, 'state.user', {})`
https://lodash.com/docs/4.17.11#get

Reply:
Good catch, I hastily copied from another section, fixed that one as well :-P